webknossos/webknossos/dataset/view.py

```diff
@@ -603,7 +558,7 @@ def __exit__(
         _value: Optional[BaseException],
         _tb: Optional[TracebackType],
     ) -> None:
-        self.close()
+        pass
```

Comment:
Please add a warnin on `__enter__`, that the view does not need to be used as a context manager anymore.